src/status_im/chat/models/images.cljs

```diff
@@ -175,13 +180,24 @@
 (rf/defn image-selected
   {:events [:chat.ui/image-selected]}
   [{:keys [db]} current-chat-id original uri]
-  {:db (update-in db [:chat/inputs current-chat-id :metadata :sending-image original] merge {:uri uri})})
+  {:db (-> db
+           (update-in [:chat/inputs current-chat-id :metadata :sending-image original] merge {:uri uri})
+           (update-in [:chat/inputs current-chat-id :metadata :selected-photos]
+                      (fn [selected-photos]
+                        (->> (conj selected-photos original)
+                             (into [])))))})
 
 (rf/defn image-unselected
   {:events [:chat.ui/image-unselected]}
   [{:keys [db]} original]
-  (let [current-chat-id (:current-chat-id db)]
-    {:db (update-in db [:chat/inputs current-chat-id :metadata :sending-image] dissoc original)}))
+  (let [current-chat-id       (:current-chat-id db)
+        selected-photos       (-> (:chat/inputs db)
```

Comment:
really hard to read, better to do not use thread macro for such forms 
```clojure
inputs (:chat/inputs db)
selected-photos (get-in inputs [current-chat-id :metadata :selected-photos])
```
more readable